delivery_carrier_label_default/models/stock.py

```diff
@@ -1,27 +1,19 @@
-# -*- coding: utf-8 -*-
-# Copyright 2013-2016 Camptocamp SA
+# Copyright 2013-2018 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
-from openerp import api, models
+from odoo import api, models
 
 
 class StockPicking(models.Model):
     _inherit = 'stock.picking'
 
     @api.multi
     def generate_default_label(self, package_ids=None):
-        """
-        Generate a label from a qweb report
-        """
+        """Generate a label from a qweb report."""
         self.ensure_one()
-        report_obj = self.env['ir.actions.report.xml']
-
-        data = {'ids': self.id}
-        report_obj = self.env['report']
-        pdf_report = report_obj.get_pdf(
-            self, 'delivery_carrier_label_default.report_default_label',
-            data=data)
-
-        return {'name': '%s.pdf' % "Shipping Label",  # report.report_name,
-                'file': pdf_report,
-                'file_type': 'pdf',
-                }
+        report = self.env.ref('delivery_carrier_label_default.default_label')
```

Comment:
note for improvement: maybe we can make this configurable at company level.
Not mandatory: if you don't want to do it now (it's a migration) you can drop a comment here for the future.

Reply:
note added

Reply:
Please put the note in ROADMAP.rst file and regenerate README.